protocol/wasmbinding/msg_plugin.go

```diff
@@ -48,6 +58,16 @@ func (m *CustomMessenger) DispatchMsg(ctx sdk.Context, contractAddr sdk.AccAddre
 		if contractMsg.DepositToSubaccount != nil {
```

Comment:
```
		if err := json.Unmarshal(msg.Custom, &contractMsg); err != nil {
			return nil, nil, errorsmod.Wrap(err, "sending msg")
		}
```

In the above code could we make the unmarshall error more descriptive? Instead of just saying `"sending msg"`

Meta: I know we are just trying unblock Elixir atm, but let's still try to make checked in code as production quality as  possible. This reduces the burden of future reviews